test/types/range/lydia/badRange-string-multi.good

```diff
@@ -0,0 +1,2 @@
+badRange-string-multi.chpl:1: In function 'main':
+badRange-string-multi.chpl:2: error: Range bounds must be of compatible types
```

Comment:
In retrospect, this error message seems a little too bad in this case since the types seem compatible (they're both strings) but just aren't types we can build ranges from.

Reply:
I'll return to the other suggested message of "Bounds of 'low..high' must be integers of compatible types." (which matches the message in other places, iirc)

Reply:
Err, maybe "Range bounds of 'low..high' ..."?

Reply:
Or I could just add "integers" in "must be of compatible" section (shoulda thought of that first)